packages/module-react/package.json

```diff
@@ -1,5 +1,21 @@
 {
   "name": "@wxt-dev/module-react",
+  "description": "WXT module to enable React support",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wxt-dev/wxt/tree/main/packages/module-react"
+  },
+  "homepage": "https://wxt.dev",
```

Comment:
Thoughts on having the modules point to their README instead of the main docs website? There isn't any docs specific to these modules on it.

https://github.com/wxt-dev/wxt/tree/main/packages/module-react/README.md

Reply:
Think that's better yeah